docs/resources/disable_legacy_dbfs_setting.md

```diff
@@ -0,0 +1,35 @@
+---
+subcategory: "Settings"
+---
+
+# databricks_disable_legacy_dbfs Resource
```

Comment:
name of the resource is incorrect: it should be `databricks_disable_legacy_dbfs_setting`